src/Aspire.Hosting/ApplicationModel/CommandsConfigurationExtensions.cs

```diff
@@ -8,6 +8,8 @@ namespace Aspire.Hosting.ApplicationModel;
 
 internal static class CommandsConfigurationExtensions
 {
+    private static readonly TimeSpan s_timeout = TimeSpan.FromSeconds(20);
```

Comment:
How did you pick this number?

Reply:
I picked it myself. It seems like 20 seconds is a good amount of time for a timeout.